lib/morphix.ex

```diff
@@ -90,6 +92,8 @@ defmodule Morphix do
   @spec atomorphiform!(map()) :: map()
   @spec atomorphiform!(map(), :safe) :: map()
   @spec atomorphiform!(map(), list()) :: map()
+  @spec stringomorphiform!(map) :: map()
```

Comment:
I can't believe I didn't notice this, but `stringomorphiform!` and `stringmorphify!`... the naming doesn't match. I tested the code otherwise, this is honestly the last change I have before merge. I like the `stringm` over `stringo` but could go either way.

Reply:
No worries buddy - consistency is important - I've fixed and pushed changes - thanks for sticking with it.